modules/support_ticket/config/schema/support_ticket.schema.yml

```diff
@@ -0,0 +1,86 @@
+# Schema for the configuration files of the support ticket module.
+
+support_ticket.settings:
+  type: config_object
+  label: 'Support ticket settings'
+  mapping:
+    use_admin_theme:
+      type: boolean
+      label: 'Use admin theme when editing or creating tickets'
+
+support_ticket.type.*:
+  type: config_entity
```

Comment:
We could already inherit from node now, but given we want to make this generic, not sure that makes sense.

Also as a generic thing:

We can do the generalization in little steps.